features/hq/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,28 @@
+World Helpers
+
+Given(/^I am a new, authenticated user$/) do
+  create_admin_user
+  login
+end
+
+Given(/^I am an unauthenticated user$/) do
+  logout
+end
+
+When(/^I (?:go to|am on) the "([^"]*)" page for the "([^"]*)" role$/) do  |page, role|
```

Comment:
Now that we get to rebuild our entire Cucumber suite, let's have a discussion about whether it should use such low-level steps. I tend to think that Cucumber has more value when used at a [very high level of abstraction](http://lizkeogh.com/2011/03/04/step-away-from-the-tools/) (scroll to 'Write fewer scenarios').